File: AppAPI/Models/Wishlist.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const Wishlist = new Schema({
    wishlistItem: 
    {
        type: String,
        required: true
    },
    itemDescription: 
    {
        type: String,
        required: true
    }
});
mongoose.model('Wishlist',Wishlist);

File: lib/Models/Wishlist.dart
class Wishlist {
  final String id;
    final String wishlistPoster;
  final String wishlistItem;
  final String itemDescription;

  Wishlist._(this.id,this.wishlistPoster,this.wishlistItem,this.itemDescription);

  factory Wishlist.fromJson(Map json) {
    final id = json[''].replaceAll('ObjectId(\"', '').replaceAll('\")', '');
    final wishlistPoster = json['wishlistPoster'];
    final wishlistItem = json['wishlistItem'];
    final itemDescription = json['itemDescription'];

    return Wishlist._(id,wishlistPoster,wishlistItem,itemDescription);
  }
}